lib/tracker/handlers/google_analytics.rb

```diff
@@ -36,6 +36,22 @@ def event(name:, event_args: {})
       }
     end
 
+    # Accepts:
+    #   id         - String
+    #   event_args - Hash, optional
+    #
+    # Returns:
+    #   Hash
+    def conversion(id:, event_args: {})
```

Comment:
perhaps it would be good to make conversion common method to handlers, just like `page` and `event` are now; most analytic services have some kind of 1st class support for conversion events